### PR TITLE
PUBDEV-7334: Make sure Generic Model predictions are not double-post-processed

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

**Comment by michalkurka** [comment]
@Pscheidl tests to follow

**Comment by Pscheidl** [comment]
Can't see any caveat ATM. LGTM.

File: h2o-r/tests/testdir_jira/runit_pubdev_7334_generic_scoring.R [hunk 1]
```diff
@@ -0,0 +1,22 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+
+
+test.pubdev_7334 <- function() {
+    cc_h2o <- h2o.importFile('http://h2o-public-test-data.s3.amazonaws.com/smalldata/jira/pubdev_7334.csv')
```
Comment (Pscheidl) [inline h2o-r/tests/testdir_jira/runit_pubdev_7334_generic_scoring.R]: Shold it be part of smalldata ?

File: h2o-algos/src/test/java/hex/generic/GenericModelTest.java [hunk 2]
```diff
@@ -732,10 +732,17 @@ public void downloadable_mojo_glm_binomial() throws IOException {
...
+        assertNotNull(model);
+        assertFalse(model.needsPostProcess());
+        return model;
+    } 
+    
     private Key<Frame> importMojo(final String mojoAbsolutePath) {
         final ArrayList<String> keys = new ArrayList<>(1);
-        H2O.getPM().importFiles(mojoAbsolutePath, "", new ArrayList<String>(), keys, new ArrayList<String>(),
-                new ArrayList<String>());
+        H2O.getPM().importFiles(mojoAbsolutePath, "", new ArrayList<>(), keys, new ArrayList<>(),
```
Comment (Pscheidl) [inline h2o-algos/src/test/java/hex/generic/GenericModelTest.java]: Thanks !

File: h2o-algos/src/main/java/hex/generic/GenericModelOutput.java [hunk 3]
```diff
@@ -180,12 +183,16 @@ private ModelMetrics determineModelmetricsType(final MojoModelMetrics mojoMetric
...
+    public double defaultThreshold() {
+        return _defaultThreshold;
+    }
 
     @Override
     public ModelCategory getModelCategory() {
         return _modelCategory; // Might be calculated as well, but the information in MOJO is the one to display.
     }
-
+
```
Comment (Pscheidl) [inline h2o-algos/src/main/java/hex/generic/GenericModelOutput.java]:
```suggestion

```

Nit: remove the spaces

File: h2o-r/tests/testdir_jira/runit_pubdev_7334_generic_scoring.R (hunk 1 above)
Comment (Pscheidl) [inline h2o-r/tests/testdir_jira/runit_pubdev_7334_generic_scoring.R]:
```suggestion
    cc_h2o <- h2o.importFile(locate('smalldata/jira/pubdev_7334.csv'))
```

**Pull request merged**